### PR TITLE
Gh 444/use spikes if already exist

### PR DESCRIPTION
**Pull request by sgratiy**
# Addresses:
Add an optional write_spikes with the default in the run_feature_extractor set to True
but in the run_from_nwb_file the default is set to False
this would avoid making changes to LIMS. Users processing dandi files would by default have write_spikes=False.

Addresses issue [#444]

# Type of Fix:

- [x] Bug fix (non-breaking change which fixes an issue)

# Solution:
Do not add spikes processing module if it already exists

### CODE REVIEW

**Comment by MatthewAitken** [comment]
Does this need a Lims update to write spikes in the job now?

File: ipfx/nwb_append.py [hunk 1]
```diff
@@ -47,19 +47,26 @@ def append_spike_times(input_nwb_path: PathLike,
...
+                            unit='seconds',
+                            data=wrapped_spike_times,
+                            name=f"Sweep_{sweep_num}")
+            spike_module.add_data_interface(ts)
+
+        nwbfile.add_processing_module(spike_module)
+
+        nwb_io.write(nwbfile)
+    else:
+        raise ValueError("Cannot add spikes times to the nwb file: "
```
Comment (NileGraddis) [inline ipfx/nwb_append.py:68]: do you need to raise a new valueerror here?

Comment (NileGraddis) [reply ipfx/nwb_append.py:68]: vs. just reraise (maybe with some added info)

**Comment by sgratiy** [comment]
@MatthewAitken I have changed the logic to avoid changes in LIMS. Thanks

**Pull request merged**